packages/components/src/docs/General/11-FAQ.mdx

```diff
@@ -78,12 +78,43 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
 
```

Comment:
Could we structure the page in another way so that we don’t see all questions in the vertical tab navigation on the right hand side?
Would be better if we see only:
General questions
Technical questions
![Uploading Bildschirmfoto 2024-09-20 um 12.37.15.png…]()

Reply:
That vertical nav is not very useful atm. If we had 3-4 categories instead of "General" and "Technical" this would be a nice way to navigate between them though. I'll look into it and have someone assist me with the Markup.